<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTestWorldZone</class>
 <widget class="QWidget" name="FormTestWorldZone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>217</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>188</red>
        <green>187</green>
        <blue>185</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>188</red>
        <green>187</green>
        <blue>185</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>188</red>
        <green>187</green>
        <blue>185</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>441</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="cboxCheck">
      <property name="text">
       <string>isCheck</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cboxNegate">
      <property name="text">
       <string>isNegate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cboxDisable">
      <property name="text">
       <string>isDisable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cboxEnable">
      <property name="text">
       <string>isEnable</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>index</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtIndex"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>e_index</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEntityIndex"/>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>add/sub</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtAddSub"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAddCyl">
      <property name="text">
       <string>Add_Cyl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAddBox">
      <property name="text">
       <string>Add_Box</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSetEnable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>3</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>SetEnable/Disable</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="btnGetBox">
      <property name="text">
       <string>GetBox</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGetCyl">
      <property name="text">
       <string>GetCyl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSetCheck">
      <property name="text">
       <string>CheckOn/Off</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveAll">
      <property name="text">
       <string>SaveAll</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
